code/lib/cli/src/generate.ts

```diff
@@ -34,20 +34,20 @@ const command = (name: string) =>
     .command(name)
     .option(
       '--disable-telemetry',
-      'disable sending telemetry data',
+      'Disable sending telemetry data',
       // default value is false, but if the user sets STORYBOOK_DISABLE_TELEMETRY, it can be true
       process.env.STORYBOOK_DISABLE_TELEMETRY && process.env.STORYBOOK_DISABLE_TELEMETRY !== 'false'
     )
     .option('--debug', 'Get more logs in debug mode', false)
-    .option('--enable-crash-reports', 'enable sending crash reports to telemetry data');
+    .option('--enable-crash-reports', 'Enable sending crash reports to telemetry data');
 
 command('init')
   .description('Initialize Storybook into your project.')
   .option('-f --force', 'Force add Storybook')
   .option('-s --skip-install', 'Skip installing deps')
   .option('--package-manager <npm|pnpm|yarn1|yarn2>', 'Force package manager for installing deps')
   .option('--use-pnp', 'Enable pnp mode for Yarn 2+')
-  .option('-p --parser <babel | babylon | flow | ts | tsx>', 'jscodeshift parser')
+  .option('-p --parser <babel | babylon | flow | ts | tsx>', 'Jscodeshift parser')
```

Comment:
@valentinpalkovic, one small item: can you revert `Jscodeshift` back to how it was? In this case, it was correct, the package is named as such and we should preserve it